crates/starknet-types-core/Cargo.toml

```diff
@@ -44,6 +46,7 @@ serde = ["alloc", "dep:serde"]
 prime-bigint = ["dep:lazy_static"]
 num-traits = []
 papyrus-serialization = ["std"]
+zeroing = ["zeroize"]
```

Comment:
for this types of feature that just depend on one single crate, I tend to name the feature by the crate name. Here `zeroize` rather than `zeroing`. Wdyt?